community/modules/scheduler/pbspro-client/variables.tf

```diff
@@ -0,0 +1,312 @@
+}
+
+variable "pbs_client_rpm_url" {
+  description = "Path to PBS Pro Client Host RPM file"
+  type        = string
+}
+
+variable "name_prefix" {
+  description = "Name prefix for PBS execution hostnames"
+  type        = string
+  default     = null
+}
+
+variable "instance_count" {
+  description = "Number of instances"
+  type        = number
+  default     = 1
+}
+
+variable "network_storage" {
+  description = "An array of network attached storage mounts to be configured."
+  type = list(object({
+    server_ip             = string,
+    remote_mount          = string,
+    local_mount           = string,
+    fs_type               = string,
+    mount_options         = string,
+    client_install_runner = map(string)
+    mount_runner          = map(string)
+  }))
+  default = []
+}
+
+variable "instance_image" {
+  description = "Instance Image"
+  type = object({
+    family  = string,
+    project = string
+  })
+  default = {
+    family  = "hpc-centos-7"
+    project = "cloud-hpc-image-public"
+  }
+}
+
+variable "disk_size_gb" {
+  description = "Size of disk for instances."
+  type        = number
+  default     = 200
+}
+
+variable "disk_type" {
+  description = "Disk type for instances."
+  type        = string
+  default     = "pd-standard"
+}
+
+variable "auto_delete_boot_disk" {
+  description = "Controls if boot disk should be auto-deleted when instance is deleted."
+  type        = bool
+  default     = true
+}
+
+variable "local_ssd_count" {
+  description = "The number of local SSDs to attach to each VM. See https://cloud.google.com/compute/docs/disks/local-ssd."
+  type        = number
+  default     = 0
+}
+
+variable "local_ssd_interface" {
+  description = "Interface to be used with local SSDs. Can be either 'NVME' or 'SCSI'. No effect unless `local_ssd_count` is also set."
+  type        = string
+  default     = "NVME"
+}
+
+variable "enable_public_ips" {
+  description = "If set to true, instances will have public IPs on the internet."
+  type        = bool
+  default     = true
+}
+
+variable "machine_type" {
+  description = "Machine type to use for the instance creation"
+  type        = string
+  default     = "c2-standard-60"
+}
+
+variable "labels" {
+  description = "Labels to add to the instances. List key, value pairs."
+  type        = any
+}
+
+variable "service_account" {
+  description = "Service account to attach to the instance. See https://www.terraform.io/docs/providers/google/r/compute_instance_template.html#service_account."
+  type = object({
+    email  = string,
+    scopes = set(string)
+  })
+  default = {
+    email = null
+    scopes = ["https://www.googleapis.com/auth/devstorage.read_write",
+      "https://www.googleapis.com/auth/logging.write",
+      "https://www.googleapis.com/auth/monitoring.write",
+      "https://www.googleapis.com/auth/servicecontrol",
+      "https://www.googleapis.com/auth/service.management.readonly",
+    "https://www.googleapis.com/auth/trace.append"]
```

Comment:
Odd formatting by tflint here. No need to fix it, but I'm wondering if putting the square brackets on their own lines would fix this?

Feel free to disregard.

Reply:
This format made its way into a couple places in other areas. Addressed on HEAD of PR